### PR TITLE
texture etc2 eac

### PR DESCRIPTION
**Pull request by shrekshao**
- use ktx format and hard-coded offset for the header data

### CODE REVIEW

File: samples/texture_etc2_eac.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+        // -- Load compressed texture data
+        var IMAGE_SIZE = {
+            width: 512, 
+            height: 512
+        };
+        var KTX_HEADER_OFFSET = 68;
+        
+        var xhr = new XMLHttpRequest();
+        xhr.open('GET', '../assets/img/Di-3d.ktx', true);
```
Comment (pjcozzi) [inline samples/texture_etc2_eac.html:131]: How did you generate the .ktx file?  Include a link to the tool in a comment.

File: samples/texture_etc2_eac.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        xhr.responseType = 'arraybuffer';
+        xhr.onload = function(e) {
+            if (this.status != 200) {
+                return;
+            }
+            
+            var tmpArrayBuffer = new Uint8Array(this.response);
+            var pixels = tmpArrayBuffer.subarray(KTX_HEADER_OFFSET, KTX_HEADER_OFFSET+IMAGE_SIZE.width * IMAGE_SIZE.height);
```
Comment (pjcozzi) [inline samples/texture_etc2_eac.html:139]: Include a link to the KTX file spec in case developers want to see what is in the header that we are skipping.

**Comment by pjcozzi** [comment]
Just those minor comments, this looks great!

**Comment by shrekshao** [comment]
Ready

**Comment by pjcozzi** [comment]
Thanks!

**Pull request merged**